### DISCUSSION TITLE
How to read large number of records when using opensearch sql

### DISCUSSION THREAD

**Discussion by subbareddydagumati** [aws/aws-sdk-pandas#1393] [Q&A]
How to read large number of records(more than 10k) when using opensearch "search_by_sql"

**Comment by kukushking** [comment]
I think you might be better off using [awswrangler.opensearch.search](https://aws-data-wrangler.readthedocs.io/en/stable/stubs/awswrangler.opensearch.search.html#awswrangler.opensearch.search) and `scroll` parameter. Unfortunately at the moment `search_by_sql` does not support paging

**Comment by subbareddydagumati** [comment]
Yes. I am using search. It's working fine. Can I submit enhancement request.

**Reply by kukushking** [reply]
Yes, absolutely!
